src/modules/hash.js

```diff
@@ -38,10 +38,11 @@ export class Hash {
         }
 
         this.lastHash = location.hash;
+        this.boundSync = this.sync.bind(this); //Store a bound sync wrapper for future use.
```

Comment:
Linter is complaining about the line length, enforcing 80 chars in this project:
https://travis-ci.org/koalyptus/TableFilter/builds/144638612#L366
You can place the comment above the line if you want.